app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/MockNavigationActivity.java

```diff
@@ -125,7 +129,15 @@ protected void onCreate(Bundle savedInstanceState) {
       .navigationNotification(customNotification)
       .build();
 
-    navigation = new MapboxNavigation(this, Mapbox.getAccessToken(), options);
+    //  TODO: For metrics hook test
+    MapboxMetricsReporter metricsReporter = MapboxMetricsReporter.INSTANCE;
+    metricsReporter.setMetricsObserver(this);
+    navigation = new MapboxNavigation(
+            this,
+            Mapbox.getAccessToken(),
+            options,
+            metricsReporter
```

Comment:
While i understand the intent here, we might want to keep access to the MetricsReporter internally vs. injecting via constructor parameter. It should not matter in modular arch.